cmd/server_test.go

```diff
@@ -1843,6 +1843,8 @@ func (s *TestSuiteCommon) TestGetPartialObjectMisAligned(c *C) {
 	1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,
 	1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,
 	1234567890,1234567890,1234567890,123`
+	// ensuring that the buffer contains atleast 13 bytes of data.
+	buffer.WriteString(line[:13])
```

Comment:
Repeatability comes from setting a seed value. Each different seed value, causes a different random sequence, but the keeping the seed constant causes the same sequence of random numbers to be generated.